upgrade/directories.go

```diff
@@ -393,8 +407,9 @@ func Verify5XTablespaceDirectories(tsLocations []string) error {
 			}
 
 			path := filepath.Join(tsLocation, dbOidDir.Name(), PGVersion)
-			if !PathExists(path) {
-				mErr = errorlist.Append(mErr, newTablespaceDirectoryError("5X source cluster", "missing "+path))
+			exist, err := PathExist(path)
+			if !exist {
+				mErr = errorlist.Append(mErr, newTablespaceDirectoryError("5X source cluster", "missing "+path), err)
```

Comment:
Need to check err...

```suggestion
			exist, err := PathExist(path)
			if err != nil {
							mErr = errorlist.Append(mErr, xerrors.Errorf("checking path %q: %w", path, err))
			}
			
			if !exist {
				mErr = errorlist.Append(mErr, newTablespaceDirectoryError("5X source cluster", "missing "+path), err)
```